server/api/resources.js

```diff
@@ -4,6 +4,33 @@ let session = driver.session();
 const router = require('express').Router()
 const recordsReducer = require('./records-reducer.js')
 
+router.get(`/:resourceName/reviews`, async (req,res,next) => {
+  try{
+    const resourceName = req.params.resourceName
+    const query = 
+    `
+      MATCH(u:User)-[:REVIEWS]->(rev:Review)-[:REVIEWS]->(r:Resource)
+      WHERE r.name={resourceName}
+      RETURN r.name AS resource,
+             u.name AS author,
+             rev.comments AS comments,
+             rev.score AS score
+      ORDER BY score
+      limit 3
```

Comment:
Not sure if we want to `LIMIT` here

Reply:
As mentioned regarding the reviews, I was just pulling the top 3 reviews.   Reviews are a whole new can of worms so I do't want to really touch this.  Really reviews probably didn't need to be part of this PR